### PR TITLE
Qualmless issue116

### PR DESCRIPTION
**Pull request by qualmless**
addressed #116 - There are lint errors in Curriculum and LearningResources components 

- [ ] src/app/resources/curriculum/curriculum.component.spec.ts:10:14
WARNING: 10:14 deprecation async is deprecated: use waitForAsync(), (expected removal in v12)
- [ ] src/app/resources/learning-resources/learning-resources.component.spec.ts:10:14
WARNING: 10:14 deprecation async is deprecated: use waitForAsync(), (expected removal in v12)
- [ ] src/app/resources/learning-resources/learning-resources.component.ts:12:3
ERROR: 12:3 typedef expected call-signature: 'ngOnInit' to have a typedef

### CODE REVIEW

File: src/app/resources/learning-resources/learning-resources.component.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export class LearningResourcesComponent implements OnInit {
 
   constructor() { }
 
-  ngOnInit() {
+  ngOnInit() : void {
```
Comment (alisaduncan) [inline src/app/resources/learning-resources/learning-resources.component.ts:12]: Please remove the extra space before the colon as this introduces a new lint error. You can run `npm run lint` and verify no lint errors display for the files listed in this issue. Thanks!

**Review by alisaduncan — COMMENTED** [review]
One change requested as noted. Looks like you're getting the hang of git forking workflow!! 🎉

**Review by qualmless — COMMENTED** [review]
does pushing change this automatically @alisaduncan?

**Review by alisaduncan — APPROVED** [review]

**Comment by alisaduncan** [comment]
fixes #116

**Pull request merged**